server/index.js

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable no-undef */
+const express = require('express');
+const bodyParser = require('body-parser');
+const path = require('path');
+const app = express();
+const port = 3000;
+
+app.use(bodyParser.urlencoded({ extended: false }));
+app.use(bodyParser.json());
+app.use(express.static('public'));
+
+app.get('/:courseNumber', (req, res) => {
+  res.sendFile(path.resolve('./public/index.html'));

```

Comment:
Do you really need this if you already have line 10 ?

Reply:
Ignore, on second thought I think I understand what you're trying to do here.

Reply:
I think I tried without it, but I'll try again after the merge.

If I recall correctly, without this endpoint, the proxy doesn't respond when you go to localhost:3000/2.  Static will pick it up if you go to localhost:3000/, but we're trying to render courses so we need to know which course to render.  In this case, I'm still going to need the static route for the stylesheet for the proxy.